system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -43,6 +44,74 @@
+             response (ingest/ingest-concept concept {:token user1-token})]
+        (is (= 200 (:status response)))
+        (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))
+     (testing "ingest on PROV1 with subscription request for subscriber-id below subscription limit"
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-7"
+                                                                   :Name "sub2"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+        (is (= 201 (:status response)))
+        (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))))
+
+(deftest subscription-count-exceeds-limit-test
+  (side/eval-form `(jobsub/set-subscriptions-limit! 1))
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-1"
+                 :EntryTitle "entry-title-exceed-limit-1"})
+               {:token "mock-echo-system-token"})
+        user1-token (echo-util/login (system/context) "user1")]
+     (testing "ingest on PROV1 with subscription request for subscriber-id below subscription limit" ;zzz
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-7"
+                                                                   :Name "sub1"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+         (is (= 201 (:status response)))
+         (is (mdb/concept-exists-in-mdb? (:concept-id response) (:revision-id response)))))
+     (testing "ingest on PROV1 with subscription request for subscriber-id already at subscription limit"
+       (let [concept (subscription-util/make-subscription-concept {:Query "platform=NOAA-9"
+                                                                   :Name "sub2"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"})
+             response (ingest/ingest-concept concept {:token user1-token})]
+        (is (= 409 (:status response)))
+        (is (= "The subscriber-id [user1] has already reached the subscription limit." (first (:errors response))))))
+    (side/eval-form `(jobsub/set-subscriptions-limit! 100))
+    (testing "ingest on PROV1 with subscription request for subscriber-id below updated subscription limit"
```

Comment:
I'm not sure this test provides value.  

When the limit is 1, you proved... 

1.  it succeeds at ingesting 1 subscription
2. it succeeds when you have 1 subscription with multiple revisions
3. it fails when you try to create more subscriptions than the limit.

Could probably get rid of it.

Reply:
You could add a test to prove that after hitting the limit and then deleting a subscription, that a user can then create a new subscription

Reply:
I think we intended this last test to validate that updating `subscriptions-limit` to 100 was actually changing the limit and allowing the user to create more subscriptions. If you think it should still be removed I can get rid of it though.